services/docker/docker-version.tester.js

```diff
@@ -0,0 +1,46 @@
+'use strict'
+
+const { isSemVer } = require('../test-validators')
+const t = (module.exports = require('../tester').createServiceTester())
+
+t.create('docker version (valid, library)')
+  .get('/_/alpine.json')
+  .expectBadge({
+    label: 'version',
+    message: isSemVer,
+  })
+
+t.create('docker version (valid, library with tag)')
+  .get('/_/alpine/latest.json')
+  .expectBadge({
+    label: 'version',
+    message: isSemVer,
+  })
+
+t.create('docker version (valid, user)')
+  .get('/jrottenberg/ffmpeg.json')
+  .expectBadge({
+    label: 'version',
+    message: isSemVer,
+  })
+
+t.create('docker version (valid, user with tag)')
+  .get('/jrottenberg/ffmpeg/3.2-alpine.json')
+  .expectBadge({
+    label: 'version',
+    message: isSemVer,
+  })
+
+t.create('docker version (invalid, incorrect tag)')
+  .get('/_/alpine/wrong-tag.json')
+  .expectBadge({
+    label: 'version',
+    message: 'no tags found',
+  })
+
+t.create('docker version (invalid, unknown repository)')
+  .get('/_/not-a-real-repo.json')
+  .expectBadge({
+    label: 'version',
+    message: 'invalid response data',
```

Comment:
Same comment as https://github.com/badges/shields/pull/4562/files#r377397815

Reply:
Fixed.

Reply:
Also it's worthwhile calling out that this cannot reside in transform for the version badge because of the additional lookup, so it resides in handle.